tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py

```diff
@@ -25,13 +26,16 @@
     'bits'
 ]
 
+ACTIVATION_QUANTIZATION_MODES = ['activations', 'outputs']
+QUANTIZATION_MODES = ['weights'] + ACTIVATION_QUANTIZATION_MODES
```

Comment:
I would call it QUANTIZABLE_ENTITIES. Maybe there is a better name but QUANTIZATION_MODES is really confusing now not here but below in the code.